fs/localfs/local_fs.go

```diff
@@ -141,6 +141,14 @@ func toDirEntryOrNil(dirEntry os.DirEntry, prefix string) (fs.Entry, error) {
 	return entryFromDirEntry(fi, prefix), nil
 }
 
+func (fsd *filesystemDirectory) IterateEntries(ctx context.Context, cb func(context.Context, fs.Entry) error) error {
+	if err := fs.ReaddirToIterate(ctx, fsd, cb); err != nil {
```

Comment:
nit: in wrapping is not that convenient in this particular case, it would also be ok to do `// nolint:wrapcheck` or even add a rule to exclude this in `.golangci.yml`